.github/workflows/full_suite_integration_tests_cron.yml

```diff
@@ -0,0 +1,191 @@
+          echo "**Job run:** [${{ github.run_id }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})" >> message.txt
+          echo "**Build scan URL:** ${{ needs.run-it-full-suite.outputs.build-scan-url }}" >> message.txt
+          echo "**Commits since the last run:**" >> message.txt
+          list=$(curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/commits?since=${{ steps.last-run-date.outputs.last-run-date }}" | jq ' .[] | .sha' -r)
+          if [ -z "$list" ]
+          then
+            echo "No new commits" >> message.txt
+          else
+            echo "|commit SHA|PR|" >> message.txt
+            echo "|---|:---:|" >> message.txt
+            while IFS= read -r commit; do
+              resp=$(curl -H "Accept: application/vnd.github.groot-preview+json" -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/commits/${commit}/pulls" | jq '.[] | {html_url, title}')
+              if [ -z "$resp" ]; then
+                echo "|${commit}|-|" >> message.txt
+              else
+                echo "|${commit}|[PR-$(echo $resp | jq .title -r)]($(echo $resp | jq .html_url -r))|" >> message.txt
+              fi
+            done <<< "$list"
+          fi
+
+      - name: Create success message
+        if: ${{ needs.run-it-full-suite.outputs.it-status == 'success' }}
+        run: |
+          echo "### Full suite IT run :white_check_mark: SUCCEEDED :white_check_mark:" >> message.txt
+          echo "**Timestamp:** ${{ steps.current-time.outputs.formattedTime }}" >> message.txt
+          echo "**Job run:** [${{ github.run_id }}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})" >> message.txt
+          echo "**Build scan URL:** ${{ needs.run-it-full-suite.outputs.build-scan-url }}" >> message.txt
+          echo "**Closing issue**" >> message.txt
+
+      - name: Make message body
+        id: body
+        run: |
+          body=$(cat message.txt)
+          body="${body//'%'/'%25'}"
+          body="${body//$'\n'/'%0A'}"
+          body="${body//$'\r'/'%0D'}"
+          echo "::set-output name=body::${body}"
+
+      - name: Decide on next step
+        id: next-step
+        run: |
+          next_step=''
+          status=${{ needs.run-it-full-suite.outputs.it-status }}
+          issue_number=${{ steps.check-opened.outputs.issue-number }}
+          if [ "$status" == 'success' ]; then
+            if [ "$issue_number" == null ]; then
+              next_step='do-nothing'
+            else
+              next_step='close'
+            fi
+          else
+            if [ "$issue_number" == null ]; then
+              next_step='create'
+            else
+              next_step='comment'
+            fi
+          fi
+          echo "::set-output name=next-step::${next_step}"
```

Comment:
as we discussed, please make task to rewrite it to `flank-scripts` because this is not so readable

Reply:
also when we will move it to `flank-scripts` it will be easier to test
```
get the current time
check if the issue is already opened (sets output issue-number)
get time when the workflow was started last time, skip cancelled and in_progress (sets output last-run-date)
create either a success or failure message
for failure message:
get a list of commits since last-run-date
check if the list is empty
for each commit get associated PR
append the results table
make comment body (we need to escape newline characters)
decide what to do next:
if IT status is success and no issue opened -- do nothing
if IT status is success and there is an issue opened -- add a comment and close the issue
it IT status is failure and there is no issue opened -- create an issue and add comment
if IT status is failure and there is issue opened -- add comment
```

Reply:
Personally, I think that we should move whole this logic to flank-scripts in this PR

Reply:
My intention was to do it in a separate issue, but I think I can make it there if you wish

Reply:
> also when we will move it to flank-scripts it will be easier to test

Good point, will do it